### PR TITLE
More font features

### PR DESCRIPTION
**Pull request by denisjackman**
adding in more fonts and font features

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Jackmanimation/XaviGame/pull/2" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ I will do this in C++
 ========
 * [TileEd](http://www.mapeditor.org/)
 * [tinyxml](http://www.grinninglizard.com/tinyxml/)
+* [Fonts](https://www.fontsquirrel.com)
```
Comment (denisjackman) [inline README.md]: Should this be in the main game project ?

Comment (denisjackman) [inline README.md]: !fix

Comment (denisjackman) [inline README.md]: !resolve

Comment (denisjackman) [inline README.md]: !fixed

**Pull request merged**